TwitchLib.Api/Models/Helix/Analytics/GetGameAnalyticsResponse.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace TwitchLib.Api.Models.Helix.Analytics
+{
+    public class GetGameAnalyticsResponse
+    {
+        [JsonProperty(PropertyName = "data")]
+        public GameAnalytics[] GameId { get; protected set; }
```

Comment:
maybe rename this to GameAnalytics or Data instead of GameId?

Reply:
oops